web-api/src/lib.rs

```diff
@@ -23,7 +22,8 @@
     clippy::pedantic,
     noop_method_call,
     rust_2018_idioms,
-    unused_qualifications
+    unused_qualifications,
+    unsafe_op_in_unsafe_fn
```

Comment:
There is a false positive related to macro hygenics and encapsulation AFIKT  this falls over a `#[deny(unsafe_op_in_unsafe_fn)]` which fails as `deny` is less strict then `forbid` and it's set in
the `std::thread_local!` macro (`src/std/thread/local.rs.html#185`) used by `tokio::task_local!`.